frontend/www/js/omegaup/components/course/Form.vue

```diff
@@ -3,7 +3,7 @@
     <div v-if="!update" class="card-header">
       <h3 class="card-title">{{ T.courseNew }}</h3>
```

Comment:
```suggestion
      <h3 class="card-title mb-0">{{ T.courseNew }}</h3>
```

Así, de una vez solucionamos lo del espacio debajo del texto en el título del card.